README.md

```diff
@@ -0,0 +1,235 @@
+# Katzenpost Mix Network
+
+## Post Quantum Anonymous Communication Network
+
+![build badge](https://github.com/katzenpost/katzenpost/actions/workflows/go.yml/badge.svg?branch=main)
+
+Katzenpost is a software project dedicated to designing and
+implementing mix network protocols. A mix network is a type of
+anonymous communication network. An anonymous communication
+network is also known as a traffic analysis resistant network; that is,
+it's protocols are designed to resist statistical analysis by passive
+global adversaries.
+
+Traffic analysis typically refers to the statistical analysis of
+encrypted traffic. Traffic analysis is worth defending against given
+that common network protocols leak lots of information such as source
+and destination IP addresses, message size, message sequence, message
+delay pattern, geographical locations, social graph etc. Mere end to
+end encryption alone cannot protect against this type of information
+leakage.
+
+At the most basic level, mixnets are composed of mix nodes. These are
+a cryptographic packet switching routers which protect our privacy by
+mixing many messages together and cryptographically transforming them
+before routing them on to the next hop. Mix nodes also use shuffling
+or added latency to create uncertainty for network observers. This
+uncertainty is in regards to trying to link incoming messages with the
+outgoing messages.
+
+
```

Comment:
we can also link to https://katzenpost.network/ and put a project status page there

Reply:
https://github.com/katzenpost/katzenpost/issues/358